db/migrate/20180606143538_create_bus_stop_fields.rb

```diff
@@ -0,0 +1,9 @@
+class CreateBusStopFields < ActiveRecord::Migration[5.1]
+  def change
+    create_table :bus_stop_fields do |t|
```

Comment:
To continue our discussion @dfaulken, why is this wrong?

Reply:
I'm not fighting for this being right, I just don't know why it's wrong.

Reply:
It's not a big deal since you're using an Active record model for this anyway. Conventionally join tables will be the two table names, both pluralized, in alphabetical order.

But since the model is called BusStopField, arguably it's also convention for the table to be named bus stop fields.

So just removing the create join table statement is probably fine.